libraries/aws/shadow/integration-test/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+include("../shadowFilePaths.cmake")
+include(${MODULES_DIR}/standard/mqtt/mqttFilePaths.cmake)
+project ("shadow system test")
+cmake_minimum_required (VERSION 3.13)
+
+# ====================  Define your project name (edit) ========================
+set(project_name "shadow_system")
+
+# ================= Create the library under test here (edit) ==================
+
+# list the files you would like to test here
+list(APPEND real_source_files
+            ${SHADOW_SOURCES}
+        )
+# list the directories the module under test includes
+list(APPEND real_include_directories
+            .
+            ${SHADOW_INCLUDE_PUBLIC_DIRS}
+            ${LOGGING_INCLUDE_DIRS}
+        )
+
+# =====================  Create UnitTest Code here (edit)  =====================
+
+# list the directories your test needs to include
+list(APPEND test_include_directories
+            .
+            ${SHADOW_INCLUDE_PUBLIC_DIRS}
+            ${LOGGING_INCLUDE_DIRS}
+        )
+
+# list the directories your test needs to include
+list(APPEND test_include_directories
+            .
+            ${MQTT_INCLUDE_PUBLIC_DIRS}
+            ${MQTT_INCLUDE_PRIVATE_DIRS}
+            ${LOGGING_INCLUDE_DIRS}
+        )
+
+# =============================  (end edit)  ===================================
+set(real_name "${project_name}_real")
+
+create_real_library(${real_name}
+                    "${real_source_files}"
+                    "${real_include_directories}"
+                    # Empty mock name as create_real_library needs the 4th argument.
+                    ""
+        )
+
+list(APPEND stest_link_list
+            lib${real_name}.a
+        )
+
+list(APPEND stest_dep_list
+            ${real_name}
+            mqtt
+            clock_posix
+            openssl_posix
+        )
+
+set(stest_name "${project_name}_test")
+set(stest_source "${project_name}_test.c")
+create_test(${stest_name}
+            ${stest_source}
+            "${stest_link_list}"
+            "${stest_dep_list}"
+            "${test_include_directories}"
+        )
+
+# Download the Mosquitto Root CA certificate.
+message( "Downloading the Mosquitto Root CA certificate..." )
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/certificates)
+execute_process(
+    COMMAND curl --url https://test.mosquitto.org/ssl/mosquitto.org.crt
+    -o ${CMAKE_CURRENT_LIST_DIR}/certificates/mosquitto.org.crt
+)
```

Comment:
It seems that AmazonRootCA1.crt for this integration test and is already downloaded by the demos. 
Please remove downloading of Mosquitto Root CA certificate.

Reply:
yes, you are right, will remove it.

Reply:
will remove it!